### PR TITLE
Auth pages

### PR DESCRIPTION
**Pull request by fwojtowicz**
I've created login and signup views, which are **not** connected to backend yet.

### CODE REVIEW

File: client/src/components/LoginPage.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+<script>
+import Navbar from "./Navbar";
+
+export default {
+  name: "SignUpPage",
+  components: { Navbar },
+  data() {
+    return {
+      activePath: this.$route.path == "/login"
```
Comment (BartoszGamza) [inline client/src/components/LoginPage.vue:29]: what is that supposed to do?

File: client/src/components/SignUpPage.vue [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  <div>
+    <Navbar />
+    <div class="container">
+      <div class="signup-form-container">
+        <form>
+          <input class="input-textarea" type="text" placeholder="E-mail" />
+          <input class="input-textarea" type="text" placeholder="Passsword" />
+          <input class="input-textarea" type="text" placeholder="Confirm password" />
+          <div class="signup-form-buttons">
+            <button class="signup-form-button" :class="{'active-theme' :activePath}">Register</button>
```
Comment (BartoszGamza) [inline client/src/components/SignUpPage.vue:11]: this is super reduntant, could you make one SignUp component which sould support the login and register modes? This is why it was designed this way, to limit the unecessary copying

File: client/src/components/LoginPage.vue (hunk 1 above)
Comment (BartoszGamza) [reply client/src/components/LoginPage.vue:29]: data is static by definition, this should be a computed

File: client/src/components/SignUpPage.vue [hunk 3]
```diff
@@ -26,55 +31,75 @@ export default {
...
-    };
+    return {};
+  },
+
+  computed: {
+    activePathIsSignUp() {
+      return this.$route.path == "/signup";
+    },
+    primaryButton() {
+      if (this.activePathIsSignUp) {
```
Comment (BartoszGamza) [inline client/src/components/SignUpPage.vue:42]:
```js
return this.activePathIsSignUp ? 'Register' : 'Log in'
```
WDYT?

Comment (fwojtowicz) [reply client/src/components/SignUpPage.vue:42]: wunderbar

**Pull request merged**